Dockerfile

```diff
@@ -40,6 +40,8 @@ WORKDIR /app
 
 COPY --chown=${UID}:${GID} --from=maven target/value-added-service-*.jar app.jar
 
+RUN apk update && apk upgrade --no-cache libcrypto3 libssl3
```

Comment:
Modifying an image with `apk update` and `apk upgrade` is not allowed as it will change the dependencies that was already scanned and reported by DockerHub. If there is a library that is out of date/contains vulnerability contact the creator of the base image (eclipse-temurin) to update their image.

Reply:
Removed the docker file command and as talked when a new version of the docker image is released he should update it